test/check-paths.sh

```diff
@@ -37,8 +37,7 @@ exactly 26 "text2path uses" 'text2path'
 exactly 3 "update_icon() override" '/update_icon\((.*)\)'  -P
 exactly 1 "goto uses" 'goto '
 exactly 468 "spawn uses" 'spawn\s*\(\s*(-\s*)?\d*\s*\)' -P
-exactly 3 "obj/New uses" '^/obj.*/New\('
-exactly 0 "relatively pathed /New calls" '\tNew\('
```

Comment:
As said on discord, pls use `^\s+([A-z0-9_]+/)*New\(.*\).*$` with an exactly of 1 instead of just deleting this.

Reply:
Whoop, bash is interpreting that `$` as a var or something, use:
`^\s+([A-z0-9_]+/)*New\(.*\).*\$`

Reply:
Above don't work with the bash script in the unit test environment or locally on wsl/ubuntu/bash. Can add in a separate PR if you have a working alternative.